package.json

```diff
@@ -1,49 +1,53 @@
 {
```

Comment:
remove all edits to the package.json file, all folder dependencies should be added at folder level and not the global scope

Reply:
Create the pckg.json in your folder and work with it from there